src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -223,38 +226,51 @@ function MoneyRequestConfirmPage(props) {
-                        selectedParticipants={participants}
-                        iouAmount={props.iou.amount}
-                        iouComment={props.iou.comment}
-                        iouCurrencyCode={props.iou.currency}
-                        onConfirm={createTransaction}
-                        onSendMoney={sendMoney}
-                        onSelectParticipant={(option) => {
-                            const newParticipants = _.map(props.iou.participants, (participant) => {
-                                if (participant.accountID === option.accountID) {
-                                    return {...participant, selected: !participant.selected};
-                                }
-                                return participant;
-                            });
-                            IOU.setMoneyRequestParticipants(newParticipants);
-                        }}
-                        receiptPath={props.iou.receiptPath}
-                        receiptSource={props.iou.receiptSource}
-                        iouType={iouType.current}
-                        reportID={reportID.current}
-                        // The participants can only be modified when the action is initiated from directly within a group chat and not the floating-action-button.
-                        // This is because when there is a group of people, say they are on a trip, and you have some shared expenses with some of the people,
-                        // but not all of them (maybe someone skipped out on dinner). Then it's nice to be able to select/deselect people from the group chat bill
-                        // split rather than forcing the user to create a new group, just for that expense. The reportID is empty, when the action was initiated from
-                        // the floating-action-button (since it is something that exists outside the context of a report).
-                        canModifyParticipants={!_.isEmpty(reportID.current)}
-                        policyID={props.report.policyID}
-                        bankAccountRoute={ReportUtils.getBankAccountRoute(props.report)}
-                        iouMerchant={props.iou.merchant}
-                        iouCreated={props.iou.created}
-                    />
+                    {/*
+                     * The MoneyRequestConfirmationList component uses a SectionList which uses a VirtualizedList internally.
+                     * VirtualizedList cannot be directly nested within ScrollViews of the same orientation.
+                     * To work around this, we wrap the MoneyRequestConfirmationList component with a horizontal ScrollView.
+                     */}
+                    <ScrollView>
+                        <ScrollView
+                            horizontal
+                            contentContainerStyle={[styles.flex1, styles.flexColumn]}
+                        >
```

Comment:
I don't see the problem or the need for the double ScrollView. The VirtualizedList does not seem to be directly nested. Can you please double checked whether this is needed?

Reply:
@s77rt It is needed because `MoneyRequestConfirmationList` uses `SectionList`. From [the docs](https://reactnative.dev/docs/sectionlist):
> This is a convenience wrapper around VirtualizedList, and thus inherits its props (as well as those of ScrollView) that aren't explicitly listed here...

Running the app on iOS/Android will throw a runtime error and may also break up some functionality as seen in this video:  

<video src="https://github.com/Expensify/App/assets/56603839/32741238-7598-435c-8379-97dad93b858d" width="250px">

Reply:
Ah I see you are testing with a vertical ScrollView. What I meant is to use one horizontal ScrollView.

Reply:
My bad here. We need a vertical scrollview. Thanks for your quick responses!

Reply:
Unfortunately, we can't. The `horizontal` prop of the child `ScrollView` must be removed together with the parent `ScrollView`, otherwise the whole page won't be scrollable.

Reply:
@s77rt
> Thanks for your quick responses!

No problem! Sorry for the delay, I was out of town over the weekend and just got home. I would have replied sooner if I could, sorry again!